packages/suite/src/components/wallet/CoinmarketFooter/index.tsx

```diff
@@ -130,6 +136,10 @@ const CoinmarketFooter = () => {
                         </Text>
                     </FooterBox>
                 )}
+                <Link href={URLS.TOS_INVITY_URL} target="_blank" variant="nostyle">
```

Comment:
```suggestion
                <Link href={URLS.TOS_INVITY_URL} variant="nostyle">
```
`target="_blank"`  is a default value of `Link` component.
i see its used everywhere (line 123, line 118) pls remove it as well